sales/sale_amazon/apply.rst

```diff
@@ -0,0 +1,124 @@
+   access keys. As that form depends on the data protection policy of the region of your seller
+   account (e.g. GDPR in Europe), we cannot provide you with a pre-filled form. Instead, the answers
+   of questions related to Odoo are listed in the `Answer the Additional Form`_ section.
+
+.. image:: ./media/dev_form.png
+
+Answer the Additional Form
+==========================
+
+This section lists all questions asked by Amazon in additional forms. The answers are tailored for
+*Odoo Online* and *Odoo.sh*. If you did not receive any additional form after your :ref:`initial
+application for Amazon MWS keys <amazon/developer-form>`, disregard the following.
+
+
+.. warning::
+   If you host Odoo yourself (*on-premise*), adapt the answers related to hosting to your own
+   infrastructure and data protection policy.
+
+
+- **Describe all functionalities in your application where Personally Identifiable Information (e.g.
+  customer name, street address, billing address) is required.**
+
+  - If you did not apply for the *Merchant Fulfilled Shipping* function:
+
+    \- Generation of customer invoices
+
+  - If you applied for the *Merchant Fulfilled Shipping* function:
+
+    | - Generation of delivery orders
+    | - Generation of customer invoices
+
+- **List all outside parties with whom your organization shares Amazon Information (e.g. information
+  exposed by Amazon through Amazon MWS, Seller Central, or Amazon's public-facing websites) and
+  describe how your organization shares this information.**
+
+  - If you do not share Amazon Information with outside parties:
+
+    Odoo does not share any information with outside parties.
+
+  - If you share Amazon Information with outside parties:
+
+    [Description of your organization's policy regarding Amazon Information]
+
+- **List all non-Amazon MWS sources where you retrieve Amazon Information.**
+
+  Odoo only relies on MWS to retrieve Amazon Information.
+
+- **Describe how your organization restricts public access to databases, file servers, and
+  desktop/developer endpoints.**
+
+  TODO
+
+- **Describe how your organization uniquely identifies employees and restricts access to Amazon
+  Information on a need-to-know basis.**
+
+  TODO or organization dependent ?
+
+- **Describe how your organization prevents Amazon Information from being accessed from employee
+  personal devices.**
+
+  Odoo does not prevent employees from accessing the organization's data from personal devices.
+  Role-based restrictions and access rights still apply.
+
+- **Provide details on your organization's privacy and data handling policies (a link to your policy
+  is also acceptable).**
+
+  [Description of your organization's privacy and data handling policies]
+
+- **Describe where your organization stores Amazon Information and provide details on how you
+  encrypt this information (e.g., algorithm).**
+
+  TODO storage
```

Comment:
To check with saas office:
- amazon information is stored in unencrypted database. Direct access to the database is not possible for the customer outside of UI interactions or API calls, granular access rights control ensure access is not shared to all users of the database.